pypeit/core/tracewave.py

```diff
@@ -244,12 +241,29 @@ def trace_tilts_work(arcimg, lines_spec, lines_spat, thismask, slit_cen, inmask=
         sub_img = arcimg_trans[min_spat:max_spat, :]
         sub_inmask = inmask_trans[min_spat:max_spat,:]
         sub_thismask = thismask_trans[min_spat:max_spat,:]
-        if do_crude: # First time tracing, do a trace crude
-            tilts_guess_now, err_now = trace_slits.trace_crude_init(
-                sub_img, np.array([lines_spec[iline]]), (sub_img.shape[0] - 1) // 2, invvar=sub_inmask, radius=fwhm,
-                nave=tcrude_nave, maxshift0=tcrude_maxshift0, maxshift=tcrude_maxshift, maxerr=tcrude_maxerr)
-            tilts_guess_now=tilts_guess_now.flatten()
-        else:  # A guess was provided, use that as the crutch, but determine if it is a full trace or a sub-trace
+        if do_crude:
+            # First time tracing, do a trace crude
+
+            # TODO: Worried that using a starting row that is blindly
```

Comment:
Please remove this comment. I had to think about it, but you'll note the image is sub-imaged about each lines spatial position, see min_spat, and max_spat. So the midpoint of the sub-image is the spatial location of the arc line at the center of the slit and this is exactly the behavior we want.

Reply:
Removed.